src/Sensor.cc

```diff
@@ -64,7 +64,8 @@ const std::vector<std::string> sensorTypeStrs =
   "navsat",
   "segmentation_camera",
   "boundingbox_camera",
-  "custom"
+  "custom",
+  "wideanglecamera"
```

Comment:
We should document this string here: https://github.com/ignitionrobotics/sdformat/blob/f2a286cbf89aca7daa722a70a357e0de32224372/sdf/1.9/sensor.sdf#L10-L36

Most other cameras are using `camel_case` (at least partially), can we do it here too? i.e. `wideangle_camera` or `wide_angle_camera`, or all of them to be safe?

Reply:
the existing convention with gazebo classic is [wideanglecamera](https://github.com/osrf/gazebo/blob/gazebo11/worlds/lensflare_plugin.world#L97). I think we can choose to add alternates, but I would keep `wideanglecamera` for backwards compatibility

Reply:
changed `wideanglecamera` to `wide_angle_camera` here. Also updated the Load function to check for both `wide_angle_came` and `wideanglecamera` when loading the sdf for backward compatibility. 
8aacb98